packages/interactive_media_ads/lib/src/ads_request.dart

```diff
@@ -11,10 +11,53 @@ class AdsRequest {
   AdsRequest({
     required String adTagUrl,
     ContentProgressProvider? contentProgressProvider,
+    bool? adWillAutoPlay,
+    bool? adWillPlayMuted,
+    bool? continuousPlayback,
+    double? contentDuration,
```

Comment:
`Duration?`?

(If these stay doubles rather than Durations, this one needs a unit naming too.